packages/ui-new/tests/ThemeHelperUtils.test.ts

```diff
@@ -0,0 +1,149 @@
+      background: '--apkt--background',
+      theme: {
+        dark: {
+          accent: '--apkt--theme-dark-accent'
+        },
+        light: {
+          accent: '--apkt--theme-light-accent'
+        }
+      }
+    })
+    expect(
+      ThemeHelperUtil.createCSSVariables({ ...styles, background: undefined }).cssVariables
+    ).toStrictEqual({
+      theme: {
+        dark: {
+          accent: '--apkt--theme-dark-accent'
+        },
+        light: {
+          accent: '--apkt--theme-light-accent'
+        }
+      }
+    })
+    expect(
+      ThemeHelperUtil.createCSSVariables({ ...styles, background: {} }).cssVariables
+    ).toStrictEqual({
+      theme: {
+        dark: {
+          accent: '--apkt--theme-dark-accent'
+        },
+        light: {
+          accent: '--apkt--theme-light-accent'
+        }
+      }
+    })
+    expect(
+      ThemeHelperUtil.createCSSVariables({ ...styles, background: {} }).cssVariables
+    ).toStrictEqual({
+      theme: {
+        dark: {
+          accent: '--apkt--theme-dark-accent'
+        },
+        light: {
+          accent: '--apkt--theme-light-accent'
+        }
+      }
+    })
+
+    expect(ThemeHelperUtil.createCSSVariables(styles).cssVariablesVarPrefix).toStrictEqual({
+      background: 'var(--apkt--background)',
+      theme: {
+        dark: {
+          accent: 'var(--apkt--theme-dark-accent)'
+        },
+        light: {
+          accent: 'var(--apkt--theme-light-accent)'
+        }
+      }
+    })
+    expect(
+      ThemeHelperUtil.createCSSVariables({ ...styles, background: undefined }).cssVariablesVarPrefix
+    ).toStrictEqual({
+      theme: {
+        dark: {
+          accent: 'var(--apkt--theme-dark-accent)'
+        },
+        light: {
+          accent: 'var(--apkt--theme-light-accent)'
+        }
+      }
+    })
+    expect(
+      ThemeHelperUtil.createCSSVariables({ ...styles, background: {} }).cssVariablesVarPrefix
+    ).toStrictEqual({
+      theme: {
+        dark: {
+          accent: 'var(--apkt--theme-dark-accent)'
+        },
+        light: {
+          accent: 'var(--apkt--theme-light-accent)'
+        }
+      }
+    })
+    expect(
+      ThemeHelperUtil.createCSSVariables({ ...styles, background: {} }).cssVariablesVarPrefix
+    ).toStrictEqual({
+      theme: {
+        dark: {
+          accent: 'var(--apkt--theme-dark-accent)'
+        },
+        light: {
+          accent: 'var(--apkt--theme-light-accent)'
+        }
+      }
+    })
+  })
+
+  it('should assign css variables based on style object', () => {
+    const styles = {
+      background: 'red',
+      theme: {
+        dark: {
+          accent: '#000'
+        },
+        light: {
+          accent: '#fff'
+        }
+      }
+    }
+
+    const { cssVariables } = ThemeHelperUtil.createCSSVariables(styles)
+
+    expect(ThemeHelperUtil.assignCSSVariables(cssVariables, styles)).toStrictEqual({
+      '--apkt--background': styles.background,
```

Comment:
Do we need to separate `apkt` prefix with double dash? Like maybe we could just use single;
```suggestion
      '--apkt-background': styles.background,
```